spiffworkflow-frontend/src/routes/DataStoreNew.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+import ProcessBreadcrumb from '../components/ProcessBreadcrumb';
+import DataStoreForm from '../components/DataStoreForm';
+import { DataStore, HotCrumbItem } from '../interfaces';
+import { setPageTitle } from '../helpers';
+
+export default function DataStoreNew() {
+  const searchParams = new URLSearchParams(document.location.search);
```

Comment:
Should this use the useSearchParams hook instead? LIke `const [searchParams] = useSearchParams();` from src/components/ProcessModelListTiles.tsx